### DISCUSSION TITLE
How to resize/modify body shapes at runtime?

### DISCUSSION THREAD

**Discussion by LittleCodingFox** [jrouwe/JoltPhysics#1103] [Q&A] [answered]
I'm thinking of situations where a body's properties are meant to be changed - for example, an expanding platform that later gets smaller. How would you handle this with Jolt?

**Comment by jrouwe** [comment, marked as answer]
Please see #700

**Reply by LittleCodingFox** [reply]
Thank you very much! Sorry for accidentally causing a duplicate, I searched for size and should have searched for scale!
